src/DynamicMvvm.Abstractions/DynamicMvvm.Abstractions.csproj

```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<TargetFrameworks>netstandard2.0</TargetFrameworks>
-		<LangVersion>10</LangVersion>
+		<LangVersion>12</LangVersion>
```

Comment:
We can remove langversion since netstandard2.0 does not support C# 12

Reply:
Where did you see this information? I didn't know.

Reply:
netstandard2.0 can be used with .net 8 but not C#12 unfortunately.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-versioning

To use C# 12 features, we have to target .net 8.